src/main/java/org/folio/bulkops/domain/bean/HoldingsRecord.java

```diff
@@ -159,7 +159,7 @@ public class HoldingsRecord implements BulkOperationsEntity, ElectronicAccessEnt
 
   @JsonProperty("administrativeNotes")
   @Valid
-  @CsvCustomBindByName(column = "Administrative notes", converter = StringListConverter.class)
+  @CsvCustomBindByName(column = "Administrative Note", converter = StringListConverter.class)
```

Comment:
why is it Note and not note? all other columns are not capitalized, I believe in CSV it will be Note as well when on UI it will be just note - no consistency at all